src/intel/RoomIntel.ts

```diff
@@ -499,6 +540,7 @@ export class RoomIntel {
 			if (room.controller && Game.time % 5 == 0) {
 				this.recordControllerInfo(room.controller);
 			}
+			this.autoPoison(room);
```

Comment:
Matt put similar stuff here during the recent mega-merge but this isn't where any of this stuff should go